### PR TITLE
uffizzi_2.3.3

### PR DESCRIPTION
**Pull request by moklidia**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/u/uffizzi.rb [hunk 1]
```diff
@@ -1,30 +1,16 @@
...
-  bottle do
-    sha256 cellar: :any,                 arm64_sonoma:   "7775093bbe792b5c8a98159d3e09294fe5cf651e67273d8f28e24cb57bc00995"
-    sha256 cellar: :any,                 arm64_ventura:  "5b48032ccdc363005e2839144d8ddaea81f85f53947cffd0dfaf24baaa1b20bc"
-    sha256 cellar: :any,                 arm64_monterey: "595a22c97103a4131cd73df11219065a4b1a5a3f3054bc1771799e1cd5dc1322"
-    sha256 cellar: :any,                 sonoma:         "aa110074e8efe4c68559785320d19975c3fbda888cd2db0d71e15ca66893324a"
-    sha256 cellar: :any,                 ventura:        "b0ea91e2ed4b9f741461a076659d92506dd31c1fbf6858226d23e77c1311fa46"
-    sha256 cellar: :any,                 monterey:       "77d7e4c37b5f6d7cd4a2867662fcd38d199670cf718ffe4ba6615902ca688ccb"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "5b65edddec7936934aedbc201fff91e7884d7fbc93ca7b1d2c0b0eaa970fca47"
-  end
-
```
Comment (chenrui333) [inline Formula/u/uffizzi.rb:22]: dont remove these, thanks! :)

Comment (moklidia) [reply Formula/u/uffizzi.rb:22]: Sure, I thought this is added automatically each time

**Review by iMichka — DISMISSED** [review]
Hi. Why did you downgrade sentry-ruby and activesupport?

**Comment by moklidia** [comment]
> Hi. Why did you downgrade sentry-ruby and activesupport?

@iMichka This was copied over by mistake from a tap. I have restored and tested the original gem versions.

**Review by p-linnane — DISMISSED** [review]
Thanks @moklidia!

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**